### PR TITLE
Fixed the problem(send attachment txt file fails)

### PR DESCRIPTION
**Pull request by leffss**
Fixed the problem in issue: https://github.com/kootenpv/yagmail/issues/242

### CODE REVIEW

File: yagmail/message.py [hunk 1]
```diff
@@ -230,6 +230,11 @@ def get_mime_object(is_marked_up, content_string, encoding):
             content_object["main_type"] = "application"
             content_object["sub_type"] = "octet-stream"
 
+    # Fixed the problem in issue: https://github.com/kootenpv/yagmail/issues/242
+    if content_object["main_type"] == 'text' and content_object["sub_type"] == 'plain':
+        content_object["main_type"] = 'application'
+        content_object["sub_type"] = 'octet-stream'
+
```
Comment (kootenpv) [inline yagmail/message.py:237]: Thanks for your contribution! 

Maybe it means we should never define the codebase then to use `text/plain`? 

I'm kinda confused how `application/octet-stream` would be good when wanting to send text?

Comment (leffss) [reply yagmail/message.py:237]: I just found inspiration here: https://stackoverflow.com/questions/9541837/attach-a-txt-file-in-python-smtplib/48417682#48417682

Comment (leffss) [reply yagmail/message.py:237]: And it work well on alimail(https://qiye.aliyun.com/)，qqmail(https://mail.qq.com), 126mail(https://mail.126.com) and gmail when sending  .txt attachment file

Comment (leffss) [reply yagmail/message.py:237]: I just upload a fork of pypi: https://pypi.org/project/yagmail2/

**Comment by kootenpv** [comment]
It's generally a bad idea to make a clone for a package for one change.

What's wrong with just fixing the original code and changing text/plain in the whole codebase to application/octet-stream rather than adding extra lines?

**Comment by leffss** [comment]
> It's generally a bad idea to make a clone for a package for one change.
> 
> What's wrong with just fixing the original code and changing text/plain in the whole codebase to application/octet-stream rather than adding extra lines?

Because this problem is random, not all mailboxes will appear; And I really don't know why change text/plain to application/octet-stream can fix it.

**Pull request closed without merging**